internal/workflow.go

```diff
@@ -400,6 +400,11 @@ type (
 		RunID string
 	}
 
+	// WorkflowExecutuonAsync Details.
+	WorkflowExecutionAsync struct {
+		ID string
```

Comment:
probably `WorkflowID` instead?  it's meaningful that it's not some other ID, and particularly that it's not a RunID.

Reply:
I agree on more explicit name such as `WorkflowID` but want to keep this similar to `WorkflowExecution` struct above. Inconsistency across similar domain objects would cause confusion for users

Reply:
ah, yeah, just barely out of diff-context view.
matching that makes sense 👍